recipes/sql-sqlline

```diff
@@ -0,0 +1,4 @@
+(sql-sqlline
+ :fetcher gitlab
+ :repo "matteo.redaelli/sql-sqlline"
+ :files ("artifacts/*.el"))
```

Comment:
I think this points to the wrong directory -- maybe you can move your .el files to the project root and use the default recipe?

Reply:
I think I removed all warnings related to  M-x package-lint-current-buffer

Reply:
But there are no `.el` files in the directory `artifacts` - your recipe needs fixing, I believe.

Reply:
You had to run "make" to see files *.el in artifacts/
They are a copy from files from "src/*el" and I did not want to save them in git
In any case now I have changed my .gitignore file and you should see files in artifacts/

I hope now it is ok